ods_ci/tests/Tests/1000__model_serving/1007__model_registry_jupyter_notebook.robot

```diff
@@ -0,0 +1,110 @@
+    Wait Until Workbench Is Started     workbench_title=registry-wb
+    Wait For Model Registry Containers To Be Ready
+    Upload File In The Workbench     filepath=${SAMPLE_ONNX_MODEL}    workbench_title=${WORKBENCH_TITLE}
+    ...         workbench_namespace=${PRJ_TITLE}
+    Upload File In The Workbench     filepath=${JUPYTER_NOTEBOOK_FILEPATH}    workbench_title=${WORKBENCH_TITLE}
+    ...         workbench_namespace=${PRJ_TITLE}
+    Launch And Access Workbench    workbench_title=${WORKBENCH_TITLE}
+    ...    username=${TEST_USER.USERNAME}     password=${TEST_USER.PASSWORD}
+    ...    auth_type=${TEST_USER.AUTH_TYPE}
+    Jupyter Notebook Should Run Successfully     ${JUPYTER_NOTEBOOK}
+
+
+*** Keywords ***
+Prepare Model Registry Test Setup
+    [Documentation]    Suite setup steps for testing Model Registry.
+    Set Library Search Order    SeleniumLibrary
+    # Skip If Component Is Not Enabled    modelregistry
+    RHOSi Setup
+    Launch Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    ...    ${ODH_DASHBOARD_URL}    ${BROWSER.NAME}    ${BROWSER.OPTIONS}
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Apply Db Config Samples    namespace=${PRJ_TITLE}
+    Fetch CA Certificate If RHODS Is Self-Managed
+
+Apply Db Config Samples
+    [Documentation]    Applying the db config samples from https://github.com/opendatahub-io/model-registry-operator
+    [Arguments]    ${namespace}
+    ${rc}    ${out}=    Run And Return Rc And Output
+    ...    oc apply -k ${MODEL_REGISTRY_DB_SAMPLES}/mysql -n ${namespace}
+    Should Be Equal As Integers	  ${rc}	 0   msg=${out}
+
+Teardown Model Registry Test Setup
+    [Documentation]  Teardown Model Registry Suite
+    JupyterLibrary.Close All Browsers
+    ${return_code}    ${output}=    Run And Return Rc And Output    oc delete project ${PRJ_TITLE} --force --grace-period=0
+    Should Be Equal As Integers	  ${return_code}	 0
+    Log    ${output}
+    RHOSi Teardown
+
+Jupyter Notebook Should Run Successfully
+    [Documentation]    Runs the test workbench and check if there was no error during execution
+    [Arguments]    ${filepath}
+    Open Notebook File In JupyterLab    ${filepath}
+    # Open With JupyterLab Menu  Run  Run All Cells
+    Open With JupyterLab Menu  Run  Restart Kernel and Run All Cells…
+    Click Element    xpath=//div[contains(text(),"Restart") and @class="jp-Dialog-buttonLabel"]
+    Wait Until JupyterLab Code Cell Is Not Active  timeout=120s
+    Sleep    2m    msg=Waits until the jupyter notebook has completed execution of all cells
+    JupyterLab Code Cell Error Output Should Not Be Visible
+    SeleniumLibrary.Capture Page Screenshot
+    Run Cell And Check For Errors    print("RegisteredModel:");print(registry.get_registered_model(registeredmodel_name))
+    SeleniumLibrary.Capture Page Screenshot
+
+Upload File In The Workbench
```

Comment:
you picked this one from `ods_ci/tests/Tests/1000__model_serving/1007__model_serving_llm/1007__model_serving_caikit_py_client.robot` didn't you?  it'd better to move the kw in a common resource file rather than duplicating the code